nixos/modules/tasks/kbd.nix

```diff
@@ -49,7 +49,7 @@ in
     })
 
     (mkIf setVconsole {
-      environment.systemPackages = [ pkgs.kbd ];
+      environment.systemPackages = with pkgs; [ kbd kbdKeymaps.dvp kbdKeymaps.neo ];
```

Comment:
Does that mean that the patchset (as is) doesn't git bisect cleanly? If so, please consider squashing / re-ordering.